GameOfLife/GameOfLife/Iteration.cs

```diff
@@ -0,0 +1,162 @@
+﻿namespace GameOfLife
+{
+    public class Iteration
+    {
+        public List<int> cellsToDieX = new List<int>();
+        public List<int> cellsToDieY = new List<int>();
```

Comment:
Would make sense to make these parameters private as they are used only within this class.

Reply:
Made all the parameters that are used within their class private.

Reply:
You can mark this comment as resolved.
For some reason can't do it myself.